packages/dashboard-core-plugins/src/panels/IrisGridPanel.tsx

```diff
@@ -93,6 +98,7 @@ type ModelQueue = ModelQueueFunction[];
 
 interface Metadata {
   table: string;
+  type?: VariableTypeUnion;
```

Comment:
This should maybe be required. Although I don't think the spot where it's passed in (`GridPlugin`) is strictly checked, but if it were in the future, we would want to make sure this is required

Reply:
Yea would need to worry about migration from older data then. On Enterprise this isn't really an issue since we know all the variable types on a query, so we just check what the variable type is for that variable name. Theoretically we could do that here as well, but it would require a round trip (calling the `subscribeToFieldUpdates` API, listening for "changes", scanning for the variable, ugh).